src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.rc

```diff
@@ -320,6 +320,13 @@ BEGIN
     CONTROL         "Alpha",IDC_TEXT_SPPM_RADIANCE_ALPHA,"CustEdit",WS_TABSTOP,106,211,18,10
     CONTROL         "Alpha",IDC_SPINNER_SPPM_RADIANCE_ALPHA,"SpinnerControl",WS_TABSTOP,125,211,6,10
     GROUPBOX        "Radiance  Estimation",IDC_STATIC,0,140,199,84
+    CONTROL         "Visualize Photons",IDC_CHECK_SPPM_VIEW_PHOTONS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,4,241,71,10
+    GROUPBOX        "Debug Options",IDC_STATIC,0,229,199,44
+    LTEXT           "Initial View Radius:",IDC_STATIC_SPPM_VIEW_PHOTONS_RADIUS,4,256,60,8
+    CONTROL         "Initial Radius",IDC_TEXT_SPPM_VIEW_PHOTONS_RADIUS,
```

Comment:
Keep the same name ("Views Radius"), especially since the word radius is heavily used around here.

Reply:
OK